tests/unit/specs/components/ActionModal/components/__snapshots__/DelegationModal.spec.js.snap

```diff
@@ -17,7 +17,7 @@ exports[`DelegationModal Submission Data for Redelegating should set the subtota
   <tmformgroup-stub fieldid="from" fieldlabel="From" class="action-modal-form-group">
     <tmfield-stub type="select" value="2" options="[object Object],[object Object],[object Object]" id="from" title="cosmosvaladdr15ky9du8a2wlstz6fpx3p4mqpjyrm5ctplpn3au"></tmfield-stub>
   </tmformgroup-stub>
-  <tmformgroup-stub fieldid="amount" fieldlabel="Amount" class="action-modal-form-group"><span class="input-suffix max-button">STAKE</span>
+  <tmformgroup-stub fieldid="amount" fieldlabel="Amount " class="action-modal-form-group"><span class="input-suffix max-button">STAKE</span>
```

Comment:
What? 👁

Reply:
I think this is because of the `(Optional)`

Reply:
fixed